domain/secret/state/state.go

```diff
@@ -416,13 +416,16 @@ VALUES ($secretID.id)
 	}
 
 	dbRevision := secretRevision{
+		ID:       revisionUUID.String(),
+		SecretID: uri.ID,
+		Revision: 1,
+	}
+	obsolete := secretRevisionObsolete{
```

Comment:
I don't think there's any need to insert a row here until the revision becomes obsolete